Configuration/HLT/python/autoCondHLT.py

```diff
@@ -1,5 +1,7 @@
 # https://cms-conddb.cern.ch/browser/#search
 
+snapshotTime = "9999-12-31 23:59:59.000"
```

Comment:
@Martin-Grunewald it is not a miracle, it is a *protection* against mistakes.
More in details, this protects you against the following issue:

1. a new L1T menu is stored to the DB, but the tag is still open to changes (synchronisation type set to `any`)
2. you run your tests, they look fine
3. the L1T developers change it by mistake with a new IOV
4. AlCa rejects the tag (if designed for our supported scenarios) because of the mistake
5. you re-run your tests, and they look different.